documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

```diff
@@ -3,38 +3,40 @@
 // // assembly-upgrade.adoc
 
 [id='assembly-upgrade-resources-{context}']
-= Strimzi resource upgrades
+= Strimzi custom resource upgrades
 
-The `v1alpha1` API version is deprecated for the following Strimzi resources:
+[role="_abstract"]
+After you have upgraded Strimzi to {ProductVersion}, you must ensure that your custom resources are using API version `{ApiVersion}`.
 
-* `Kafka`
-* `KafkaConnect`
-* `KafkaConnectS2I`
-* `KafkaConnector`
-* `KafkaMirrorMaker`
-* `KafkaMirrorMaker2`
-* `KafkaTopic`
-* `KafkaUser`
-* `KafkaBridge`
-* `KafkaRebalance`
+IMPORTANT: The upgrade of resources _must_ be performed after xref:assembly-upgrade-{context}[upgrading the Cluster Operator], so the Cluster Operator can understand the resources.
 
-The `v1beta1` API version is deprecated for the following Strimzi resources:
+.CLI upgrades to custom resources
 
-* `Kafka`
-* `KafkaConnect`
-* `KafkaConnectS2I`
-* `KafkaMirrorMaker`
-* `KafkaTopic`
-* `KafkaUser`
+Strimzi provides am _API conversion tool_ with its release artifacts
+From its CLI, you can use the tool to convert the format of your custom resources to `{ApiVersion}`:
 
-Update these resources to use the `{KafkaApiVersion}` API version.
+* xref:proc-upgrade-cli-tool-files-{context}[]
+* xref:proc-upgrade-cli-tool-direct-{context}[]
 
-This section describes the upgrade steps for the resources.
+Following on from the conversion of your custom resources, you can also set `{ApiVersion}` as the _storage_ API version in your CRDs:
 
-IMPORTANT: The upgrade of resources _must_ be performed after xref:assembly-upgrade-{context}[upgrading the Cluster Operator], so the Cluster Operator can understand the resources.
+* xref:proc-upgrade-cli-tool-crds-{context}[]
+
+.Manual upgrades to custom resources
+
+As an alternative to using the API conversion tool, you can manually update each custom resource to use `{ApiVersion}`:
```

Comment:
We should make clear here that this replaces the conversion of the CRs. But not the CRD upgrade.

Reply:
I'm assuming the tool is the only way for the CRD upgrades: _After these changes, you must use the API conversion tool to upgrade your CRDs._

Reply:
Yes, correct.